implementations.md

```diff
@@ -19,3 +19,21 @@ If you know of any associated projects that are not listed here, please file a p
 ## Testing
 
 * [huawei-openlab/oct](https://github.com/huawei-openlab/oct) - Open Container Testing framework for OCI configuration and runtime
+
+## Hooks
+
+For addressing container issues that are outside the scope of this specification, the following hooks may be useful.
+
+### Linux
+
+#### Networking
+
+* [CNI][], the Container Network Interface.
+  The [`exec-plugins.sh`][cni-exec-plugins.sh] script can be used in both [prestart][] (with `add`) and [poststop][] (with `del`) hooks insert and remove a container from networks [configured][cni-conf] in a host-side directory like `/etc/cni/net.d`.
```

Comment:
On Tue, Jan 26, 2016 at 10:54:42AM -0800, W. Trevor King wrote:
> I think this PR is on hold until someone has time to translate it into
> a wiki page [1]…

More food for an eventual wiki page: on IRC today [@crosbymichael
pointed out][1] [netns][2], which includes docs on specifically
integrating via a pre-start hook.

[1]: http://ircbot.wl.linuxfoundation.org/eavesdrop/%23opencontainers/%23opencontainers.2016-02-05.log.html#t2016-02-05T21:34:49
[2]: https://github.com/jfrazelle/netns